src/System.CommandLine.Tests/NativeAOTTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET6_0_OR_GREATER
+
+using System.CommandLine.Suggest;
+using System.CommandLine.Tests.Utility;
+using System.IO;
+using System.Text;
+using FluentAssertions;
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.CommandLine.Tests;
+
+public class NativeAOTTests
+{
+    private readonly ITestOutputHelper _output;
+    private readonly string _systemCommandLineDllPath;
+
+    public NativeAOTTests(ITestOutputHelper output)
+    {
+        _output = output;
+
+        _systemCommandLineDllPath = typeof(Command).Assembly.Location;
+    }
+
+    [ReleaseBuildOnlyTheory]
+    [InlineData("-p:IlcOptimizationPreference=Speed")]
+    [InlineData("-p:IlcOptimizationPreference=Size")]
```

Comment:
Do we need to do both? This test is just to catch warnings, right? Will it produce different warnings between these two options? If we don't need to do both, that will cut the amount of time in half.